tests/integration/plots/test_plots.py

```diff
@@ -634,3 +635,21 @@ def test_nested_x_defn_collection(tmp_dir, dvc, scm, capsys):
         {"step": 1, "Max_Leaf_Nodes": "50", "rev": "workspace"},
         {"step": 2, "Max_Leaf_Nodes": "500", "rev": "workspace"},
     ]
+
+
+def test_plots_empty_directory(tmp_dir, dvc, scm, capsys):
+    (tmp_dir / "empty").mkdir()
+
+    from dvc.utils.serialize import modify_yaml
+
+    with modify_yaml("dvc.yaml") as dvcfile_content:
+        dvcfile_content["plots"] = [
+            {"empty": {}},
+        ]
```

Comment:
```suggestion
        dvcfile_content["plots"] = [{"empty": {}}]
```

Reply:
I don't think this needs `modify_yaml` at all, as you are not modifying but creating.

```suggestion
    (tmp_dir / "dvc.yaml").dump({"plots": [{"empty": {}}]})
```